src/components/Container.js

```diff
@@ -52,15 +74,15 @@ const contentAnimationVariants = {
     transition: {
       when: 'beforeChildren',
       type: 'tween',
-      duration: 0.2,
+      duration: 0.15,
```

Comment:
if durations are used consistently, shall we move them out to the base styles?

Reply:
They aren't entirely consistent, but I can unify some